scripts/prepare_sector_network.py

```diff
@@ -2401,12 +2434,10 @@ def add_h2_reconversion_tyndp(n, spatial, nodes, buses_h2_z2, costs, options=Non
         )
 
     if options["hydrogen_fuel_cell"]:
-        logger.info("Adding Z2 hydrogen fuel cell for re-electrification.")
-        # TODO: Improve with TYNDP specific technology and cost assumptions
         n.add(
             "Link",
-            nodes.index + " H2 Z2 h2-fuel-cell",
-            bus0=nodes.country.values + " H2 Z2",
+            nodes.index + f" {suffix} Fuel Cell",
```

Comment:
I think this one got lost when merging the master. It should remain as h2-fuel-cell
```suggestion
            nodes.index + f" {suffix} h2-fuel-cell",
```